windows posix weirdness test repro

jessfraz opened the pull request:
try and get reproducable windows weirdness

tianon commented:
https://botbot.me/freenode/docker-maintainers/msg/34043368/ :wink:

tianon commented:
Especially https://botbot.me/freenode/docker-maintainers/msg/34043571/ and https://botbot.me/freenode/docker-maintainers/msg/34043667/

jessfraz committed:
try and get reproducable windows weirdness

Docker-DCO-1.1-Signed-off-by: Jessie Frazelle <princess@docker.com> (github: jfrazelle)
